__tests__/app.test.js

```diff
@@ -267,9 +267,30 @@ describe("task 10 - /api/users",()=>{
                     avatar_url:
                       'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg'
                   }
-                expect(body.users[0]).toMatchObject(desiredObj)    
+                expect(Array.isArray(body.users)).toBe(true)
+                expect(body.users[0]).toMatchObject(desiredObj)
                 expect(body.users.length).toBe(4)
             })
         })
     })
+})
+describe("task 11 - /api/articles?topic= *",()=>{
+    describe("GET QUERY - topic",()=>{
+        test("status code:200 and returns an array of articles filtered by topic",()=>{
+            return request(app).get("/api/articles?topic=cats")
+            .expect(200).then(({body})=>{
+                expect(body.articles.length).toBe(1)
+                body.articles.every((article)=>{
```

Comment:
Please use foreach here as .every() needs a truthy value to be returned from the callback, otherwise your callback runs only once.